components/function-controller/pkg/apis/serverless/v1alpha1/function_types.go

```diff
@@ -89,7 +113,7 @@ type FunctionStatus struct {
 // +kubebuilder:printcolumn:name="Running",type="string",JSONPath=".status.conditions[?(@.type=='Running')].status"
 // +kubebuilder:printcolumn:name="Version",type="integer",JSONPath=".metadata.generation"
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
-
+// +kubebuilder:storageversion
```

Comment:
it's not needed https://book.kubebuilder.io/multiversion-tutorial/api-changes.html#storage-versions

+ there needs to be 1 line of space between the markers and the struct as far as I remember